gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java

```diff
@@ -94,10 +90,21 @@ private InternalLoggingOptions(LoggingOptions options) {
     protected ExecutorFactory<ScheduledExecutorService> executorFactory() {
       return super.executorFactory();
     }
+
+    @Override
+    protected ApiCallSettings.Builder apiCallSettings() {
```

Comment:
Ummm I don't understand. Isn't InternalLoggingOptions is a private class here? Why would any method under a private class exposed to the user?